sdks/python/apache_beam/runners/dataflow/internal/apiclient_test.py

```diff
@@ -998,12 +998,12 @@ def test_interpreter_version_check_passes_with_experiment(self):
 
   @mock.patch(
       'apache_beam.runners.dataflow.internal.apiclient.sys.version_info',
-      (3, 8, 2))
```

Comment:
there is benefit to keep this aligned with the lowest supported version.

Hence question - do we drop Py3.8 support in the upcoming release or the following release? I forgot where we finished that conversation.

Reply:
No one really felt strongly either way, which is why this kept moving forward with the testing changes.

I can swing this back to 3.9 no problem

Reply:
reading https://github.com/apache/beam/blob/e00559140df6cb45989bc62356d3b512bd32a537/sdks/python/apache_beam/runners/dataflow/internal/apiclient.py#L85 we drop support in Dataflow for python 3.9 in the upcoming release.

Reply:
so if the plan was to "move tests to 3.9 now, but still support Python 3.8 for one more release", then we strayed from that plan

Reply:
this test essentially tests that Python 3.8 is still supported on Dataflow.

Reply:
If that's actually how we want to go about it then that's fine, but I'm of the opinion that if we're not testing things we're better off moving forward with removing support. I can put 3.8 back on that list for now if that's preferred and we can move this discussion to a later PR actually removing 3.8 support from beam proper

Reply:
this file was probably updated too early anyway IMO, best to leave it for the in-package removal

Reply:
> If that's actually how we want to go about it then that's fine, but I'm of the opinion that if we're not testing things we're better off moving forward with removing support. 

yeah, I am of the same opinion as you; my understanding was that this was a compromise b/w start the cleanup work now and also delay the deprecation for a little longer.